src/main.cpp

```diff
@@ -555,7 +540,7 @@ void loop()
             strOutput += str_sprintf("LED FPS: %d ", g_Values.FPS);
 
             #if USE_STRIP
-                strOutput += str_sprintf("LED Bright: %03.0f%%, LED Watts: %d, ", g_Values.Brite, g_Values.Watts);
+                strOutput += str_sprintf("LED Bright: %d, LED Watts: %d, ", g_Values.Watts, g_Values.Brite);

```

Comment:
I see you merged this PR while I was writing this, but you seem to be undoing a change here that you implemented yesterday. I wonder if that was intentional, as the change of yesterday looked like a fix to me.